rbc/remotejit.py

```diff
@@ -319,8 +319,10 @@ def local(self):
         return Caller(self.func, self.signature.local)
 
     def __repr__(self):
-        return '%s(%s, %s, local=%s)' % (type(self).__name__, self.func,
-                                         self.signature, self.local)
+        msg = (
+            f"{type(self).__name__}({self.func}, {self.signature})"
+        )
+        return msg
```

Comment:
nit:
```suggestion
        return f"{type(self).__name__}({self.func}, {self.signature})"
```
Just looks simpler to me.